react-blocks/src/styles/blocks/SplitTwoColumns.scss

```diff
@@ -0,0 +1,417 @@
+.split-two-column {
+  background-color: $black;
+  color: $white;
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  z-index: 2;
+  width: 100vw;
```

Comment:
this is breaking block render in the editor, might be better to disable it when the block is rendered inside the editor